tests/conftest.py

```diff
@@ -120,3 +155,67 @@ def app_with_notifs(app):
 def create_app():
     """Create test app."""
     return _create_app
+
+
+@pytest.fixture(scope="module")
+def base_app(create_app, app_config, request, default_handler):
+    """Base application fixture (without database, search and cache).
+
+    Scope: module.
+
+    This fixture is responsible for creating the Invenio application. It
+    depends on an application factory fixture that must be defined by the user.
+
+    .. code-block:: python
+
+        # confest.py
+        import pytest
+
+        @pytest.fixture(scope='module)
+        def create_app():
+            from invenio_app.factory import create_api
+            return create_api
+
+    It is possible to overide the application factory for a specific test
+    module, either by defining a fixture like above example, or simply setting
+    the ``create_app`` property on the module:
+
+    .. code-block:: python
+
+        # test_something.py
+
+        from invenio_app.factory import create_api
+        create_app = create_api
+
+        def test_acase(base_app):
+            # ...
+    """
+    # Use create_app from the module if defined, otherwise use default
+    # create_app fixture.
+    create_app = getattr(request.module, "create_app", create_app)
+    app_ = create_app(**app_config)
+
+    def delete_user_from_g(exception):
```

Comment:
is all this needed?

Reply:
it needed to be introduced after the changes described in the comment below, still needed, otherwise user is not logged out fully in tests